help/md/fnJC.md

```diff
@@ -3,15 +3,15 @@ fnJC
 ## title
 fnJC
 ## description
-Jukes cantor transition matrix
+Jukes Cantor rate matrix
 ## details
-Transition matrix with n amount of states in which each state has an equal
-probability of to change to any other state. The only parameter in this model
-is mu with the rate of change from any given state being mu / number of states.
+Rate matrix with n states in which each state has an equal probability of to change 
+to any other state. The rate of transition from one state to another is equal to 
+n / n-1.
 ## authors
 Jukes & Cantor 
```

Comment:
My impression is that we use the "authors" field (if at all) to say who came up with the RevBayes implementation rather than the model itself.